mule-gateway/modules/ROOT/pages/policies-custom-flex-getting-started.adoc

```diff
@@ -177,16 +177,24 @@ Name of the implementation.
 * minRuntimeVersion
 +
 A semantic version representing the first runtime version that the implementation binary is compatible with. For example, a mule binary might only be compatible from 4.3.0 onwards.
+* maxRuntimeVersion
++
+A semantic version representing the latest runtime version that the implementation binary is compatible with. For example, a mule binary might only be compatible from 4.3.0 and earlier.
++
+`supportedJavaVersions`
++
+The Java versions supported by the custom policy, either `"8"`, `"11"`, or `"17"`. You can list one or multiple versions. If `supportedJavaVersions` field is not provided, the policy supports only Java 8.
```

Comment:
This is only for the mule implementation. We should find a way to make that clear since I think this documentation article is being used for both Flex and Mule.

Reply:
We're not actually discussing flex here, it's just int the files name. I removed the one small mention of flex to make it clear.

Reply:
Great